tests/plugins/widget/deleteparagraphsclosetowidgets.js

```diff
@@ -0,0 +1,193 @@
+/* bender-tags: editor */
+/* bender-ckeditor-plugins: wysiwygarea,toolbar,forms,widget,codesnippet,undo,basicstyles,entities */
+
+'use strict';
+( function() {
+
+	var backspace = 8,
+		deleteKey = 46,
+		leftArrow = 37,
+		rightArrow = 39;
+
+	var input = (
+		'<div class="simplebox">' +
+		'<h2 class="simplebox-title">Title</h2>' +
+		'<div class="simplebox-content"><p>Content...</p></div></div>' +
+		'<p>&nbsp;</p>' +
+		'<pre><code class="language-javascript">//Hello another world!</code></pre>'
+	);
+
+	var notEmptyInput = (
+		'<div class="simplebox">' +
+		'<h2 class="simplebox-title">Title</h2>' +
+		'<div class="simplebox-content"><p>Content...</p></div></div>' +
+		'<p>Hello, World!</p>' +
+		'<pre><code class="language-javascript">//Hello another world!</code></pre>'
+	);
+
+	var result = (
+		'<div class="simplebox">' +
+		'<h2 class="simplebox-title">Title</h2>' +
+		'<div class="simplebox-content"><p>Content...</p></div></div>' +
+		'<pre><code class="language-javascript">//Hello another world!</code></pre>'
+	);
+
+	var simpleBox = {
```

Comment:
Could we use one of the ready-to-use widget plugins, like `image2` (Enhanced Image)? It also has an editable. It will save us some lines of code. Also, currently running a single test causes an error as Simplebox is only loaded in the first test, so we would kill two birds with one stone.

Reply:
No image2 is not suitable. I need a widget with editable fields (because Firefox shows a different behaviour if there are editable fields (see https://github.com/ckeditor/ckeditor4/pull/4433#issuecomment-764586816 )